maven-embedder/src/main/java/org/apache/maven/cli/MavenCli.java

```diff
@@ -144,37 +144,41 @@
     public static final File DEFAULT_GLOBAL_TOOLCHAINS_FILE =
         new File( System.getProperty( "maven.conf" ), "toolchains.xml" );
 
-    private static final String EXT_CLASS_PATH = "maven.ext.class.path";
+    public static final String EXT_CLASS_PATH = "maven.ext.class.path";
 
-    private static final String EXTENSIONS_FILENAME = ".mvn/extensions.xml";
+    public static final String EXTENSIONS_FILENAME = ".mvn/extensions.xml";
 
-    private static final String MVN_MAVEN_CONFIG = ".mvn/maven.config";
+    public static final String MVN_MAVEN_CONFIG = ".mvn/maven.config";
```

Comment:
Note that this property will likely go away with https://github.com/apache/maven/pull/602

Reply:
Reading #602, I'm not sure I agree with adding more intelligence in the launch scripts and removing it from the java code.  This will make tools embedding maven much more difficult to behave consistently with maven.